rotkehlchen/inquirer.py

```diff
@@ -371,6 +371,14 @@ def get_cached_current_price_entry(cache_key: Tuple[Asset, Asset]) -> Optional[C
 
         return cache
 
+    @staticmethod
+    def remove_cache_prices_for_asset(pairs_to_invalidate: List[Tuple[Asset, Asset]]) -> None:
+        """Deletes all prices cache that contains any asset in the possible pairs."""
+        for asset_pair in list(Inquirer()._cached_current_price):
```

Comment:
The `list()` is needed because I need a copy of the dict keys, given that I'm removing/popping.

Reply:
a true, if not you would be mutating the same thing that you are iterating. Then as for what we discussed yesterday doing `Inquirer()._cached_current_price.keys().copy()` is faster in 3.8+ than the list constructor but is something that can be done in a different PR since is really minor comment

Reply:
Oh okay